ur_controllers/include/ur_controllers/speed_scaling_state_controller.h

```diff
@@ -19,19 +19,60 @@
 //----------------------------------------------------------------------
 /*!\file
  *
- * \author  Felix Exner exner@fzi.de
- * \date    2019-04-17
+ * \author  Marvin Große Besselmann grosse@fzi.de
+ * \date    2021-02-10
  *
  */
 //----------------------------------------------------------------------
 
 #ifndef UR_CONTROLLERS_SPEED_SCALING_STATE_CONTROLLER_H_INCLUDED
 #define UR_CONTROLLERS_SPEED_SCALING_STATE_CONTROLLER_H_INCLUDED
 
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "controller_interface/controller_interface.hpp"
+
+#include "rclcpp_lifecycle/lifecycle_publisher.hpp"
+#include "rclcpp_lifecycle/node_interfaces/lifecycle_node_interface.hpp"
+#include "std_msgs/msg/float64.hpp"
+
 namespace ur_controllers
 {
-class SpeedScalingStateController
+class SpeedScalingStateController : public controller_interface::ControllerInterface
 {
+public:
+  SpeedScalingStateController();
+
+  controller_interface::InterfaceConfiguration command_interface_configuration() const override;
+
+  controller_interface::InterfaceConfiguration state_interface_configuration() const override;
+
+  controller_interface::return_type update() override;
+
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+  on_configure(const rclcpp_lifecycle::State& previous_state) override;
+
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+  on_activate(const rclcpp_lifecycle::State& previous_state) override;
+
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+  on_deactivate(const rclcpp_lifecycle::State& previous_state) override;
+
+protected:
+  bool init_sensor_data();
```

Comment:
use camelCase for self defined methods - will help CI pass

Reply:
I'm really against starting to mix two different style guides here. However the problem is solved since i removed this helper function all together

Reply:
Hold on the `camelCase` and use override (as you already did ) for any other function that you inherit from ros2_controllers/ros2_control which is `snake_case` formatted